examples/type-cast/type_cast.description

```diff
@@ -0,0 +1,6 @@
+// Type cast attempts casting a value to a target type, performing a numeric conversion if required.
```

Comment:
```suggestion
// Type cast casts a value to a target type performing a numeric conversion if required.
```

Reply:
Don't we need "attempts" since the cast could fail as explained later on.

Reply:
This is the exact text from the spec.

"A type-cast-expr casts the value resulting from evaluating expression to the type described by the type-descriptor, performing a numeric conversion if required."  

Then in the next paragraph, spec explains the failure behavior. So having just cast is enough in this case.  WDYT?

Reply:
Got it, will change to say "casts".